spec/frameSpec.js

```diff
@@ -0,0 +1,48 @@
+describe('Frame', function() {
+    beforeEach(function() {
+        frame = new Frame;
+    });
+
+    var completedFrame = function() {
+        for(let i = 1; i <=2; i ++) {
+            frame.storeRoll();
+        }
+        frame.totalRolls(2)
```

Comment:
some of your lines have semi colons and some don't - i'm still unsure of the best practice for this, as i think they don't necessarily need them in your use case, but wonder if it's worth doing one or the other for readability/consistency of code? i'd be intrigued by this best practice for my own code